task-10/js/app.js

```diff
@@ -0,0 +1,105 @@
+// Section 1
+
+console.log(1 + 2);
+console.log("apple" + "orange");
+console.log(1 + 2 + "apple");
+console.log("apple" + 1 + 2);
+console.log(1 + true);
+console.log(0 == false);
+console.log(1 === true);
+console.log(2 == "2");
+
+// Section 2
+
+// 1
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
+
+console.log(cricketPlayers);
+
+// 2
+
+console.log(cricketPlayers.shift())
+
+// 3
+
+console.log(cricketPlayers.length)
+
+// 4
+
+cricketPlayers.push("Ashok")
+console.log(cricketPlayers)
+
+// 5
+
+cricketPlayers.sort(function (a, b) {
+    return a.toLowerCase() - b.toLowerCase();
+});
+console.log(cricketPlayers);
+
+// 6
+
+cricketPlayersWithJerseyNumber = cricketPlayers.map((value) => value + " - " + Math.round(Math.random() * 100))
+console.log(cricketPlayersWithJerseyNumber);
+
+// 7
+
+cricketPlayersUpperCase = cricketPlayersWithJerseyNumber.map(value => value.toUpperCase())
+console.log(cricketPlayersUpperCase);
+
+// Section 3
+
+// 1 - Generate Numbers from 1 to 100
+
+function generateNumbers(start, end) {
+    let arr = []
+    for (let value = start; value <= end; value++) {
+        arr.push(value);
+    }
+    return arr;
+}
+
+let result = generateNumbers(1, 100)
+console.log(result)
+
+// 2 - Generate Date in Required Format
+
+function generateDateInFormat(dateObject) {
```

Comment:
This would give me the next date everytime, say like today is 23/02/2023, it gives me 24/02/2023;
and this looks complex can you simplify this or write a better comment ;
and also why can't you use this instead : https://www.w3schools.com/jsref/jsref_tolocaledatestring.asp